docs/en/observability/whats-new.asciidoc

```diff
@@ -11,4 +11,276 @@ https://www.elastic.co/blog/elastic-observability-7-6-0-released[7.6] |
+
+[discrete]
+== Stack monitoring alerts
+
+The {stack} monitoring features provide a way to keep a pulse on the health and performance of your {es} cluster.
+In previous releases, you could use Watcher and create custom alerts on the collected monitoring data.
+
+As of {minor-version}, we have added out of the box {kib} alerts to notify you of potential issues in the {stack}.
+Within {kib} the CPU threshold, Disk Usage, Memory Usage (JVM), and Missing monitoring data alerts are preconfigured
+and available for selection. The alerts are also visible alongside your existing Watcher cluster alerts in the
+Stack monitoring UI.
+
+Each alert is preconfigured based on the best practices recommended by Elastic. However, you can *Enter setup mode*
+and tailor them to meet your specific needs, including alert conditions and actions. Stack monitoring not only helps
+you create the alerts but also provides suggestions for investigative workflows to make this a powerful debugging tool.
+
+[role="screenshot"]
+image::images/stack-alerts.png[Stack monitoring alerts]
+
+[discrete]
+== APM service maps
+
+In 7.7.0, we introduced the service maps functionality into APM, and since then we have added data to make the maps
+more actionable. We added health indicators to make it easier for you to determine the health of your instrumented
+services and filtering to enable you to slice these maps based on your environments.
+
+As of {minor-version}, we are delighted to announce the general availability of service maps. This release provides
+a new layout to the maps. Curved connectors make it easier to visualize the connections between services.
+
+Also, you can drag nodes into a structure that is easier for you to understand. To minimize the noise on the maps,
+and to make them easier to navigate, multiple external service calls are automatically collapsed into groups to
+display the relationships between sets of services and external connections.
+
+[role="screenshot"]
+image::images/service-map.png[APM service maps]
+
+[discrete]
+== Service inventory view with sparkline charts
+
+In {minor-version}, we have added health indicators to the service inventory view. Powered by the APM {ml} integration,
+these health indicators help spot services that need attention quickly. In addition to this, we have added a sparkline
+chart to the inventory view to help you understand the historical performance of your service.
+
+[role="screenshot"]
+image::images/service-inventory.png[Service inventory view]
+
+[discrete]
+== Anomaly detection alerts
+
+In {minor-version}, we have made the APM {ml} integration more actionable by adding the capability to create alerts
+on this data. While static thresholds are great for creating basic alerts, they can potentially cause a lot of noise
+when the underlying data is expected to be bursty. The new anomaly detection based alert enables you to create alerts
```

Comment:
```suggestion
when the underlying data is expected to be bursty. The new anomaly detection-based alert enables you to create alerts
```

Reply:
Suggesting a dash because my brain stumbled over this sentence. An alternative edit is: "The new alert based on anomaly detection enables you...."

Reply:
I like your suggestion!